src/server/enterprise/cmds/cmds.go

```diff
@@ -300,7 +303,7 @@ func UnpauseCmd(ctx context.Context, pachctlCfg *pachctl.Config) *cobra.Command
 func PauseStatusCmd(ctx context.Context, pachctlCfg *pachctl.Config) *cobra.Command {
 	pauseStatus := &cobra.Command{
 		Short: "Get the pause status of the cluster.",
-		Long:  "Get the pause the cluster: normal, partially-paused or paused.",
+		Long:  "This command returns the pause state of the cluster: `normal`, `partially-paused` or `paused`.",
```

Comment:
You may wish to use a normal single quote rather than a backquote here, since I don’t believe that these messages are ever fed into a Markdown processor without escaping (I could be wrong!), but that’s up to you.

Reply:
they are fed into a markdown processor when the docs for the docs site get generated via cobra